learn/configuration/instance_options.md

```diff
@@ -38,7 +46,7 @@ Server is listening on: http://127.0.0.1:7700
 
 Command-line options take precedence over environment variables. If the same configuration option is specified both as a command-line option and as an environment variable, Meilisearch will use the command-line option and its respective value.
 
-**All configuration options must specify a value.** Using a command-line option or environment variable without specifying a value will throw an error and interrupt the launch process.
+Some configuration options must specify a value. Using a command-line option or environment variable without specifying a value will throw an error and interrupt the launch process.
 
 ```bash
 ./meilisearch --schedule-snapshot
```

Comment:
the example with --schedule-snapshot is not true anymore

Reply:
Agree with @curquiza. I'm not a fan of replacing "All" with "Some," though, as it is a bit vague.

Perhaps we could go with "Unless otherwise noted, all configuration options must specify a value"?

Or just delete the whole paragraph? This was necessary when users had to specify values even for options that did not really have an actual logical value (most of what we are calling flags now).